src/main/java/piglatin/PigLatinConverter.java

```diff
@@ -13,6 +18,7 @@ public static boolean isVowel(char letter) {
 	
 	public static String getInputFromCommand() {
 		Scanner scan = new Scanner(System.in);
+        System.out.println(ANSI_GREEN + "Please enter a word/sentence:" + ANSI_RESET);
```

Comment:
The getInputFromCommand() function should not print comments. This is not part of the functionality of this function, it should do what it says! Rather, this print should be made in the main() function.